trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    trimmed_name = @nameable.correct_name[0...10]
```

Comment:
- [ ] According to the project requirement name should be trimmed to a maximum of 10 characters to achieve that you need to   trimmed_name = @nameable.correct_name[0...9], kindly implement that

Reply:
Thank you so much for your valuable review. I will come up with solution